pkg/net/libp2p/key.go

```diff
@@ -27,13 +28,11 @@ func operatorPrivateKeyToNetworkKeyPair(operatorPrivateKey *operator.PrivateKey)
 		return nil, nil, fmt.Errorf("libp2p supports only secp256k1 operator keys")
 	}
 
-	// Libp2p keys are actually btcec keys under the hood.
-	btcecPrivateKey, btcecPublicKey := btcec.PrivKeyFromBytes(
-		DefaultCurve, operatorPrivateKey.D.Bytes(),
-	)
+	secpPrivateKey := secp.PrivKeyFromBytes(operatorPrivateKey.D.Bytes())
+	secpPublicKey := secpPrivateKey.PubKey()
```

Comment:
Why do we change that? I see that `btcec` still exposes the `PrivKeyFromBytes` function.

Reply:
Yep, we can use btcec instead of sect. Done in dbcc2dfc6ab514c423f55f42a04a4baf03395f3c.